core/src/main/java/org/kiwix/kiwixmobile/core/utils/StyleUtils.kt

```diff
@@ -45,7 +45,10 @@ object StyleUtils {
 
   @Suppress("DEPRECATION")
   @JvmStatic fun String?.fromHtml(): Spanned {
-    return if (VERSION.SDK_INT >= VERSION_CODES.N) Html.fromHtml(this, Html.FROM_HTML_MODE_LEGACY)
-    else Html.fromHtml(this)
+    return if (VERSION.SDK_INT >= VERSION_CODES.N) Html.fromHtml(
+      this ?: "",
```

Comment:
we are repeating this mapping of `this ?: " "`

We could move it higher up 
eg
```
return (this ?: "").let{
if(blah) Html(this, blah)
else Html(this)
```